manifests/p/Parsec/Parsec/150-84c/Parsec.Parsec.locale.en-US.yaml

```diff
@@ -2,17 +2,17 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
 
 PackageIdentifier: Parsec.Parsec
-PackageVersion: "150-82a"
+PackageVersion: "150-84c"
```

Comment:
```suggestion
PackageVersion: "150-84b"
```
This must match Control Panel.

Reply:
![image](https://user-images.githubusercontent.com/11600822/182004091-54fa886a-eb45-4d29-89de-f41211699963.png)